stream-chat-android-offline/src/test/java/io/getstream/chat/android/livedata/repository/SyncStateRepositoryTest.kt

```diff
@@ -1,23 +0,0 @@
-package io.getstream.chat.android.livedata.repository
-
-import androidx.test.ext.junit.runners.AndroidJUnit4
-import com.google.common.truth.Truth
-import io.getstream.chat.android.livedata.BaseDomainTest
-import io.getstream.chat.android.livedata.entity.SyncStateEntity
-import io.getstream.chat.android.livedata.utils.calendar
-import kotlinx.coroutines.runBlocking
-import org.junit.Test
-import org.junit.runner.RunWith
-
-@RunWith(AndroidJUnit4::class)
-internal class SyncStateRepositoryTest : BaseDomainTest() {
```

Comment:
In my opinion, every test should check that logic works correctly. This test checked that repo could write, read the same data. Actually, that repo is just a room wrapper. For me, there is no point to test the clean functions of mapping and Room driver as a storaging mechanism

Reply:
It can still make sure that we're not losing something on data type conversions, for example, but I suppose that should be covered by tests of the Converter classes.

And we don't want to test the mapping here either?

Reply:
I think mapping testing is too much as they are clean functions itself. Such tests usually don't give any benefits.